src/components/shared/page.tsx

```diff
@@ -48,14 +49,15 @@ const PageStyles = css`
     ${fontFace("Guardian Icons", new None, new None, "/public/fonts/icons.otf")}
 
     font-size: 62.5%;
-    background: white;
+    background: ${neutral[97]};
```

Comment:
What's the thinking behind this?

Reply:
Alongside the dark variant, this would allow the background color "flashing" a different color while the `body` is hidden. It's arguable whether the whole strategy is desirable or not however. Let's take some time to chat later on.